courses/free-courses-en.md

```diff
@@ -333,6 +333,7 @@
 ### Go
 
 * [A Tour Of Go](https://tour.golang.org/welcome/1)
+* [cht.sh Go Cheatsheet](https://cht.sh/go/:learn)
```

Comment:
:information_source: I'm almost sure that if is a cheatsheet it needs be moved to right listing `free-programming-cheatsheets.md` ([in your branch](https://github.com/RolloCasanova/free-programming-books/blob/cht.sh/more/free-programming-cheatsheets.md))

:question: Why `go:learn` and not merelly `go`?

Reply:
Well, I thought `/go:learn` was kind of a quick course; will move it to `free-programming-cheatsheets.md`.

`/go` just shows basic commands for Go in CLI.